src/sql/run.py

```diff
@@ -390,8 +395,17 @@ def run(conn, sql, config, user_namespace):
                 result = FakeResultProxy(cur, headers)
             else:
                 txt = sqlalchemy.sql.text(statement)
+                if config.autocommit:
+                    try:
+                        conn.session.execution_options(isolation_level="AUTOCOMMIT")
+                    except Exception as e:
+                        print(
+                            "The database driver doesn't support "
+                            "such AUTOCOMMIT execution option", "\nException: ", e
```

Comment:
let's add something at the end "running manual COMMIT command" so users know what we're doing

if my assumption is right, then looks like we're missing some unit tests

Reply:
also, it should be clear what part of the error message is from our side and which from the driver's side. I'm seeing this:

```
The database driver doesn't support such AUTOCOMMIT execution option 
Exception:  'duckdb.DuckDBPyConnection' object has no attribute 'set_isolation_level'
(duckdb.NotImplementedException) Not implemented Error: CREATE DATABASE not supported in DuckDB yet
[SQL: CREATE DATABASE new_db]
(Background on this error at: https://sqlalche.me/e/14/tw8g)
```

but would be better to have a format like this:

```
The database driver doesn't support such AUTOCOMMIT execution option 

Message from the database driver:
    Exception:  'duckdb.DuckDBPyConnection' object has no attribute 'set_isolation_level' (duckdb.NotImplementedException) Not implemented Error: CREATE DATABASE not supported in DuckDB yet
     [SQL: CREATE DATABASE new_db]
    (Background on this error at: https://sqlalche.me/e/14/tw8g)
```

Reply:
Updated the message to 
```
The database driver doesn't support such AUTOCOMMIT execution option
Perhaps you can try running a manual COMMIT command

Message from the database driver: 
	Exception: 'duckdb.DuckDBPyConnection' object has no attribute 'set_isolation_level' 

(duckdb.NotImplementedException) Not implemented Error: CREATE DATABASE not supported in DuckDB yet
[SQL: CREATE DATABASE new_db2]
(Background on this error at: https://sqlalche.me/e/14/tw8g)
```

(The last section starting from `duckdb.NotImplementedException` is out of control of this code, where we can't control the indent here, but I guess now it's better)
<img width="86" alt="Screenshot 2023-03-01 at 2 04 37 PM" src="https://user-images.githubusercontent.com/123580782/222239384-291c01a2-9181-4a27-adcf-1e31bbd5a5e5.png">